documentation/3.4.0/setup/native/stand-alone/README.md

```
@@ -0,0 +1,305 @@
+            ```
+
+        3. Enable Citus and set distribution columns for `user` database by running the `citus_setup.sh`with the following arguments.
+            ```
+             ./citus_setup.sh user postgres://postgres:postgres@localhost:9700/users
+            ```
+
+9.  **Update Cloud Credentials for Samiksha Service**
+
+    To enable full functionality, including certificate generation and report storage, you must configure cloud credentials in the Project Service environment file.
+
+        Path: ./ELEVATE-Samiksha/samiksha-service/.env
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)